monai/transforms/utility/array.py

```diff
@@ -758,3 +759,34 @@ def __call__(self, img: torch.Tensor):
 
         """
         return self.trans(img)
+
+
+class MapLabelValue:
+    """
+    Utility to map label values to another set of values.
+    For example, map [3, 2, 1] to [0, 1, 2], [3, 5, 8] -> [1, 2, 3], etc.
+
+    """
+
+    def __init__(self, orig_labels: Sequence[str], target_labels: Sequence[str]) -> None:
```

Comment:
I guess the types should be numbers, but how to deal with the type differences for example `orig_labels=[1.0, 2.0], target_labels=[1, 2]`, and `orig_labels=[1,2], target_labels=[3.1,2.2]`?  would be nice to support strings, for example for classification tasks: `['none', 'normal', 'lesion'] => [0, 1, 2]`

Reply:
Sure, updated in the latest commit.
Could you please help review it again?

Thanks.